data/moves.ts

```diff
@@ -609,7 +609,7 @@ export const BattleMovedex: {[moveid: string]: MoveData} = {
 					const moveid = moveSlot.id;
 					if (noAssist.includes(moveid)) continue;
 					const move = this.dex.getMove(moveid);
-					if (move.isZ || move.isMax) {
+					if (move.isZ || (move.isMax && !['gmaxoneblow', 'gmaxrapidflow'].includes(move.id))) {
```

Comment:
It looks as if you've been too eager with your find and replace... this (and similar occurrences) probably shouldn't have been changed.

Reply:
Nah, just an instance of me forgetting assist (and company) exist. I’ll fix when I’m in a computer